### DISCUSSION TITLE
How to run reuqst in VS Code ?

### DISCUSSION THREAD

**Discussion by agadinmd** [usebruno/bruno#1740] [Q&A] [answered]
Hello,

after I open a .bru file and select "Bruno: run request" from command palette, I erceive follwoing error:
![image](https://github.com/usebruno/bruno/assets/44611850/a7c68840-e043-4a35-8cc1-576f28b8aa27)

**Comment by Theqwertypusher** [comment]
This appears to be a [known issue](https://github.com/usebruno/bruno-ide-extensions/issues/4) in the Bruno IDE extension

**Comment by helloanoop** [comment, marked as answer]
Our new VSCode extension is live 🎉 🎉

VSCode - https://marketplace.visualstudio.com/items?itemName=bruno-api-client.bruno
Open VSX - https://open-vsx.org/extension/bruno-api-client/bruno

Now you can open your codebase, click on a .bru file, and voilà ✨ - Bruno runs seamlessly inside VSCode. This makes it even easier (and more rewarding) to keep your collections right where they belong - in your codebase.

<img width="800" height="459" alt="image" src="https://github.com/user-attachments/assets/9f29d975-a79b-4519-af23-bd11e38c5b01" />

**Reply by Hansehart** [reply]
Is it open source? I have a bunch of bugs.
